CMakeLists.txt

```diff
@@ -182,8 +182,14 @@ if(${CMAKE_GENERATOR} STREQUAL "Ninja")
     endif()
 endif()
 
+
 if(NETKET_USE_BLAS)
     find_package(BLAS REQUIRED)
+else()
```

Comment:
Isn't this a bit confusing? This way, even if I explicitly pass `NETKET_USE_BLAS=OFF`, it will still be used if found. (At least the name is misleading, with this change, the meaning is more like `NETKET_FAIL_IF_BLAS_NOT_FOUND`.)